karma.conf.js

```diff
@@ -61,27 +68,27 @@ module.exports = (config) => {
         colors: true,
         logLevel: config.LOG_INFO,
         autoWatch: true,
-        coverageReporter: {
+        // HTML test reporting.
+        htmlReporter: {
+            outputDir: "dist/reports/tests",
+            preserveDescribeNesting: true,
+            foldAll: false
+        },
+        coverageIstanbulReporter: {
+            fixWebpackSourcePaths: true,
             dir: process.env.CIRCLE_ARTIFACTS ?
                 process.env.CIRCLE_ARTIFACTS + '/coverage' :
                 "dist/reports/coverage",
-            check: {
+            reports: ['html', 'lcovonly', 'text-summary'],
+            thresholds: {
                 global: {
-                    lines: 80,
-                    excludes: ['src/plugins/plot/**/*.js']
+                    lines: 80
```

Comment:
I don't think we're this high. Set it to whatever our current coverage level is (61 or something from memory?). Basically I want circle-ci to fail if we fall below our current level.

Reply:
changed to `lines: 62`